account/src/lib.rs

```diff
@@ -146,11 +148,14 @@ impl<N: Network> FromStr for Account<N> {
 impl<N: Network> Display for Account<N> {
     /// Renders the account as a string.
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
+        display_string_discreetly(
```

Comment:
Let's add a CLI flag, such as `--discreet` to make this approach optional.

Reply:
Okay, I will add it later😊

Reply:
@howardwu Addressed